app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name:
+            str, health: int = 100,
+            hidden: bool = False
+    ) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        if self.health > 0:
+            Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{"
+                f"Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}"
+                f"}}")
+
+    @classmethod
+    def update_alive(cls) -> None:
+        cls.alive = [animal for animal in cls.alive if animal.health > 0]
+
+    @classmethod
+    def __str__(cls) -> str:
```

Comment:
The `__str__` method is typically used for instance string representation, not for class-level representation. Consider renaming this method to something more appropriate, like `alive_animals_str`, to avoid confusion.